backend/de.metas.serviceprovider/src/main/java/de/metas/serviceprovider/issue/IssueEntity.java

```diff
@@ -127,6 +134,14 @@ public void setBudgetedEffortIfNull(@Nullable final BigDecimal budgetedEffort)
 		}
 	}
 
+	public void setRoughEstimationIfNull(@Nullable final BigDecimal roughEstimation)
```

Comment:
really loud thinking: somehow i feel like i would avoid the Null term because it's implementation detail (u internally decided to consider null roughEstimation as `not already set`).
Maybe `setRoughEstimationIfNotSet`  or `setRoughEstimationIfNotAlreadySet` would be much more descriptive.